core/controllers/suggestion.py

```diff
@@ -91,6 +92,51 @@ def put(self, target_id, suggestion_id):
         self.render_json(self.values)
 
 
+class SuggestionToTopicActionHandler(base.BaseHandler):
+    """Handles actions performed on suggestions to topics."""
+
+    ACTION_TYPE_ACCEPT = 'accept'
```

Comment:
Why are we defining these separately per class? Seems like they should be global constants, maybe defined in suggestions_domain?

Reply:
Done